t/12-rakuast/statement-phaser.rakutest

```diff
@@ -34,7 +34,7 @@ subtest 'BEGIN phaser producing a literal expression works' => {
       for EVAL($ast), EVAL($deparsed);
 }
 
-subtest 'Block with CATCH/default handles exception and evalutes to Nil' => {
+subtest 'Block with CATCH/default handles exception and evaluates to Nil' => {
```

Comment:
There seems to be another ```evalutes``` in line 86. So, if you are on it, you could change that one as well.